Week_7/movies/3.sql

```diff
@@ -0,0 +1,3 @@
+SELECT title FROM movies
+WHERE year >= 2018
+ORDER BY title
```

Comment:
I would suggest to add an indent here like this:

```suggestion
SELECT title FROM movies
  WHERE year >= 2018
  ORDER BY title
```

Adapt this for all files.